flake8_quotes/__init__.py

```diff
@@ -30,13 +30,15 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
```

Comment:
I was referring to your first note, I removed the default in 2eb51c57b75966ce124a0b9283a39abc184c1a12 😁.